src/main/java/net/dv8tion/jda/api/entities/Message.java

```diff
@@ -2709,7 +2709,7 @@ public String getProxyUrl()
         @Nonnull
         public AttachmentProxy getProxy()
         {
-            return new AttachmentProxy(getProxyUrl());
+            return new AttachmentProxy(isImage() ? proxyUrl : url);
```

Comment:
Media proxy also works for videos, or any content that has a width and height.

```suggestion
            return new AttachmentProxy(width > 0 && height > 0 ? proxyUrl : url);
```